mycroft/client/enclosure/display_manager.py

```diff
@@ -22,6 +22,23 @@
 from mycroft.util.log import LOG
 
 
+# The DisplayManager provides "state" for the visual representation associated
+# with this Mycroft instance.  The current states are:
+#   ActiveSkill - The skill that last interacted with the display via the
+#                  Enclosure API.
+#
+# Currently, a wakeword sets the ActiveSkill to "wakeword", which will auto
+# clear after 10 seconds.
+#
+# A skill is set to Active when it matches an intent, outputs audio, or
+# changes the display via the EnclosureAPI()
+#
+# A skill is automatically cleared from Active two seconds after audio
+# output is spoken, or 2 seconds after resetting the disply.
+#
+# So it is common to have "" as the active skill.
```

Comment:
Feels like this should be a docstring instead of a comment block.

Reply:
Yeah, this was just notes to myself originally.  I made into a docstring.